### DISCUSSION TITLE
What are the chances of whisper-jax kind of speed improvement?

### DISCUSSION THREAD

**Discussion by mirozahorak** [ggml-org/whisper.cpp#814] [Ideas]
https://github.com/sanchit-gandhi/whisper-jax

**Comment by maccman** [comment]
Also curious

**Reply by moisestohias** [reply]
Whisper-jax is getting its speed-gain from:
+ **Batching**: 7x 
+ **JAX**: 2x
+ **TPU**: 5x
Overall speed-gain: 70x 

**Since Only google has TPUs, and JAX (XLA) must be instlled in order to use this version, you can only expect 7x speed-gain or less**

**Reply by x066it** [reply]
7x is not so bad too
